### PR TITLE
Fix integration

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

File: src/integration-tests/commonTests.js [hunk 1]
```diff
@@ -122,11 +122,11 @@ const verifyUserRole = async () => {
 			process.env.CLOUD_ENDPOINT
 		)
 
-		console.log('Calling verifyUserRole...')
-		const result = await verifyUserRole()
-		console.log('verifyUserRole result:', result)
+		// console.log('Calling verifyUserRole...')
+		// const result = await verifyUserRole()
+		// console.log('verifyUserRole result:', result)
```
Comment (adithyadinesh0412) [inline src/integration-tests/commonTests.js]: can we remove unwanted comments? @priyanka-TL

**Review by adithyadinesh0412 — CHANGES_REQUESTED** [review]
Reviewed on 6th Nov

File: src/integration-tests/commonTests.js (hunk 1 above)
Comment (priyanka-TL) [reply src/integration-tests/commonTests.js]: Yes I will do

**Review by adithyadinesh0412 — APPROVED** [review]
Approved 7th Nov 11.02AM

**Pull request merged**